app/js/components/breadCrumbs/breadCrumb.js

```diff
@@ -7,9 +7,27 @@ import { Link, browserHistory, withRouter } from 'react-router';
 class BreadCrumbComponent extends React.Component {
   constructor(props) {
     super(props);
+    this.handleRoute=this.handleRoute.bind(this)
+  }
+  handleRoute(e){
+    if(e.currentTarget.dataset.id == "search"){
```

Comment:
nitpick: it is advised to always use `===` instead of the `==` comparison operator because of javascript's coercion.